packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -176,8 +176,6 @@ export class NodeSDK {
 
     registerInstrumentations({
```

Comment:
i think it does make sense for this sdk to register its own tracer/meter provider here

Reply:
Not sure here. SDK calls `tracerProvider.register` and `metrics.setGlobalMeterProvider` so in good case thay are global and no need to forward it to `registerInstrumenation`.

if registration fails the instrumenations end up in using a non global Providers but most like the global ContexManager/Propagation.

I think it's better to use the globals everywhere, not just for ContextManager/Propagator.